packages/api/src/external/carequality/document/query-documents.ts

```diff
@@ -107,12 +110,17 @@ export async function getDocumentsFromCQ({
 
     const initiator = await getCqInitiator(patient);
 
+    let cqLinks = linksWithDqUrl;
+    if (cqManagingOrgFilter) {
+      cqLinks = await filterCqLinksByManagingOrg(cqManagingOrgFilter, linksWithDqUrl);
+    }
```

Comment:
Not a huge deal, but we can avoid using variable and use constant by just using a simple ternary.

Constants are are our friends - they're predictable and help minimize bugs. 😄